internal/storage/bucket_handle.go

```diff
@@ -300,6 +300,12 @@ func (bh *bucketHandle) ListObjects(ctx context.Context, req *gcs.ListObjectsReq
 		IncludeFoldersAsPrefixes: req.IncludeFoldersAsPrefixes,
 		//MaxResults: , (Field not present in storage.Query of Go Storage Library but present in ListObjectsQuery in Jacobsa code.)
 	}
+	err = query.SetAttrSelection([]string{"Name", "Size", "Generation", "Metageneration", "Updated", "Metadata"})
```

Comment:
We need to fetch all fields which are in gcs.MinObject . contentEncoding and crc32 are missing.

Reply:
Added.